packages/shared/src/components/sources/SourceCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import SquadMemberShortList from '../squads/SquadMemberShortList';
+import { Card } from '../cards/Card';
+import { Button } from '../buttons/Button';
+import { Connection } from '../../graphql/common';
+import { SourceMember } from '../../graphql/sources';
+
+interface SourceCardProps {
+  title: string;
+  subtitle?: string;
+  icon?: ReactNode;
+  image?: string;
+  description?: string;
+  action?: {
+    text: string;
+    onClick: () => void;
+  };
+  members?: Connection<SourceMember>;
+  membersCount?: number;
+  clampDescription?: boolean;
+}
+
+export const SourceCard = ({
+  title,
+  subtitle,
+  icon,
+  image,
+  description,
+  action,
+  members,
+  membersCount,
+  clampDescription = true,
+  ...props
+}: SourceCardProps): ReactElement => {
+  const items =
+    membersCount > 0 &&
+    members?.edges?.reduce((acc, current) => {
+      acc.push(current.node);
+      return acc;
+    }, []);
+
+  return (
+    <Card className="overflow-hidden p-0">
+      <div className="px-4 h-24 rounded-t-2xl bg-theme-bg-onion" />
```

Comment:
This is the header image part?
We need to accomodate a image there, maybe try render one until we have the code.
Also will ask for a fallback image to design